extras/Build/CMake/JUCEUtils.cmake

```diff
@@ -689,7 +689,16 @@ function(_juce_add_resources_rc source_target dest_target)
         ${dependency}
         VERBATIM)
 
-    target_sources(${dest_target} PRIVATE "${resource_rc_file}")
+    ###################################################################################################################
+    # Modified by minimal audio to fix standalone windows builds. The RC compiler does not support response files so
+    # we put the resource file in it's own library with no additional includes to avoid the win cmd character limit
+    if (NOT TARGET ${source_target}_rc_lib)
+        add_library(${source_target}_rc_lib OBJECT ${resource_rc_file})
+        set_target_properties(${source_target}_rc_lib PROPERTIES INCLUDE_DIRECTORIES "")
+    endif ()
+
+    target_link_libraries(${dest_target} PRIVATE ${source_target}_rc_lib)
+    ###################################################################################################################
```

Comment:
This doesn't exactly match #1, but I've realized that's because I was making edits to try and match the cmake to `7.0.9`. We'll need to address this when we do update, but for now this is what existed in our repo before. See [minimal_core/#2618](https://github.com/Minimal-Audio/minimal_core/pull/2618/files)